compiler/damlc/tests/daml-test-files/DamlDocHoogle.EXPECTED.hoogle

```diff
@@ -0,0 +1,57 @@
+
+module DamlDocHoogle
+
+-- | View docs
+
+data View
+
+
+View :: View
+
+
+getController :: I -> Party
+
+-- | T docs
+
+data T
+
+-- | T docs
+
+T :: Template T => Party -> T
+
+-- | TCall docs
+
+data TCall
+
+-- | TCall docs
+
+TCall :: Int -> TCall
+
+[i] :: TCall -> Int
```

Comment:
I think the brackets are a bit confusing here, we could pretend it's just the getter `i :: TCall -> Int`

Reply:
Yeah I don't put these in, and I'm not entirely sure why they are there, I'll have a look